### ISSUE TITLE
Error setting metadata for a project

### ISSUE THREAD

**Issue by JuanCarlosM** [GoogleCloudPlatform/google-cloud-python#4408]
Error setting metadata for a project

Debian 9.1
Python 2.7.13

How to reproduce:
> gcloud compute project-info add-metadata --metadata key=value --project=XXXXXX
**ERROR: gcloud crashed (DecodeError): Invalid enum value "INTERCONNECTS"**

If you would like to report this issue, please run the following command:
  gcloud feedback

To check gcloud for common problems, please run the following command:
  gcloud info --run-diagnostics

> gcloud info --run-diagnostics
Network diagnostic detects and fixes local network connection issues.
Checking network connection...done.                                                                                                                                                                               
Reachability Check passed.
Network diagnostic (1/1 checks) passed.

> pip show google-cloud
Name: google-cloud
Version: 0.30.0
Summary: API Client library for Google Cloud
Home-page: https://github.com/GoogleCloudPlatform/google-cloud-python
Author: Google Cloud Platform
Author-email: googleapis-publisher@google.com
License: Apache 2.0
Location: /usr/local/lib/python2.7/dist-packages
Requires: google-cloud-firestore, google-cloud-trace, google-cloud-vision, google-cloud-language, google-api-core, google-cloud-datastore, google-cloud-translate, google-cloud-error-reporting, google-cloud-pubsub, google-cloud-dns, google-cloud-runtimeconfig, google-cloud-resource-manager, google-cloud-logging, google-cloud-monitoring, google-cloud-bigtable, google-cloud-spanner, google-cloud-speech, google-cloud-videointelligence, google-cloud-bigquery, google-cloud-core, google-cloud-storage

**Comment by dhermes** [comment]
@JuanCarlosM The `gcloud` CLI is not part of this project, sorry for the confusion.

Here are some helpful links:

- https://issuetracker.google.com/issues/new?component=187143&template=0
- https://issuetracker.google.com/savedsearches/559662
- https://cloud.google.com/support/docs/issue-trackers
- https://cloud.google.com/support/docs/groups

**Comment by Benjamintf1** [comment]
Hey, I don't know if this helps, but we were using the cli to update metadata in a project using cli version 168.0.0-0. Upgraded to 182.0.0-0, and after the update got 
`No change requested; skipping update for`
Do you already have that property set in your project?
